go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/buildbuddy-io/buildbuddy
 
-go 1.21
+go 1.21.7
```

Comment:
Without this, I would get

```
diff --git a/go.mod b/go.mod
index 7b4b50bc5..744723379 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,8 @@
 module github.com/buildbuddy-io/buildbuddy

-go 1.21.7
+go 1.21.4
+
+toolchain go1.21.7

 replace (
        github.com/awslabs/soci-snapshotter => github.com/buildbuddy-io/soci-snapshotter v0.0.8 // keep in sync with buildpatches/com_github_awslabs_soci_snapshotter.patch
```

when running `bazel run //:go -- mod tidy -e`. So I pin it to the specific patch version.